scripts/image-builder/build.sh

```diff
@@ -366,6 +403,34 @@ if ${EMBED_CONTAINERS} ; then
         >> blueprint_v0.0.1.toml
 fi
 
+# Configure registry mirror
+if [ -n "${MIRROR_REGISTRY_HOST}" ] ; then
+    cat < "${SCRIPTDIR}"/config/registries.conf.template | \
```

Comment:
Having the automation is useful, but the steps managed by this command line flag are going to need to be spelled out for users to do some other way in production settings. Can we have that text in a simple how-to doc in this repo, so it's easier to forklift it to the official user documentation?

Reply:
I'd be happy to write the instructions for the official documentation. A how-to section would be a duplication in this case, similar to https://github.com/ggiguash/microshift/blob/main/docs/howto_offline_containers.md document.
Anyway, if we agree to do this, can this be done in a separate task / PR?